test/test_sgg.py

```diff
@@ -198,3 +197,11 @@ def test_secret_manager():
 def test_list_to_cloud_storage():
     response = cloud_utils.list_to_cloud_storage('sgg-test-bucket', JSON_LIST_EXAMPLE, 'test.json', timestamp=False)
     assert response == 'File test.json uploaded to sgg-test-bucket.'
+
+def test_get_customers():
+    username = 'mfutch78@gmail.com'
```

Comment:
i can continue to use your credentials if i run this on my own?

Reply:
Yep, it works because it is using google secrets